### ISSUE TITLE
[nodemon] app crashed (OSX 11.3.1 M1)

### ISSUE THREAD

**Issue by rdurnin** [ljlm0402/typescript-express-starter#118]
### Describe the Bug (버그 설명)

I have been trying to run the server in dev mode and to attach to the docker image. In both cases the app crashes and gives no clear record of why. I have found issues with bcrypt on M1 (ARM), and tried both downgrading to bcrypt 5.0.0 and replacing with bcryptjs (which does not require compiling native code). 

Tests pass in both cases without reporting errors.

### Version to Reproduce (현재 사용한 버전)

I am using typescript-express-starter@6.2.1 with node version 16.8.0, but have also tried 14.17.6 (which has issues with sequelize on start).

### Steps to Reproduce (재현 순서)

nvm use 16.8.0
npm install
npm run dev

If applicable, add a link to a test case (as a zip file or link to a repository we can clone).
-->

### Expected Behavior (예상 동작)

open localhost:3000 and see the running server

### Actual Behavior (실제 동작)

This message is in the console

[nodemon] 2.0.12
[nodemon] to restart at any time, enter `rs`
[nodemon] watching path(s): src/**/* .env
[nodemon] watching extensions: js,ts,json
[nodemon] starting `ts-node -r tsconfig-paths/register --transpile-only src/server.ts`
[nodemon] app crashed - waiting for file changes before starting...

### Additional Context (추가 사항)

The docker image builds, and I can see running logs from the mysql service.

### Notes

I am happy to help debug or bugfix in any way I am able. Please let me know, as this project is very useful to the work I am doing.

**Comment by rdurnin** [comment]
In follow up, I have tried running the server without nodemon and it also exists silently (with and without --transpile only).

./node_modules/.bin/ts-node -r tsconfig-paths/register --log-error --transpile-only src/server.ts

**Comment by ljlm0402** [comment]
@rdurnin 

Thanks for leaving the issue.

At the time of initial development, while researching the Bcrypt and Crypto modules, I adopted the Bcrypt module because it uses the Blowfish algorithm, which is very expensive for hashing, but I thought it was safe. However, I think it would be okay to use the Crypto sha512 encryption method, which is the Node basic module.

We will correct it and reflect it soon.
thank you.

**Comment by rdurnin** [comment]
Thanks for replying, and the translator was clear to understand.

The bcrypt error is coming from M1 silicone on Mac computers and is the result of compatibility with the compiled (node-gyp wasm) code. I was able to make an easy fix by doing the following:

npm import bcryptjs

(in the files using bcyrpt)

import * as bcrypt from 'bcryptjs';

The bcryptjs module runs a little (~30%) slower, but is pure js and does not require the compiled code. I can accept it as a solution until there is an ARM version of bcrypt available ([https://github.com/kelektiv/node.bcrypt.js/issues/868](url)).

The rest of the issues that I was having were probably coming from node 16(.8.0), and disappeared after I downgraded to 14(.17.6).

Thanks for following up, and for creating this package. It has been very helpful for the work I am currently doing.
